app/controllers/rents_controller.rb

```diff
@@ -0,0 +1,19 @@
+class RentsController < ApplicationController
+  include Wor::Paginate
+
+  before_action :authenticate_user!, only: %i[create index]
+
+  def create
+    rent = Rent.create(params[:rent])
```

Comment:
What if the creation fails? You will be returning a 201 to the frontend 😱 You always have to be sure about what you are informing to the consumer of your API. 
A good approach to address this problem is to instantiate the resource, try to save it, and depending on that result return a 201 or another status code with an explanation of what went wrong (take a look at [this](http://guides.rubyonrails.org/active_record_validations.html#working-with-validation-errors) about how to work with errors).
For example
```ruby
def create
  rent = Rent.new(create_params) # PARAMETERS

  if rent.save
    render json: rent, status: :created
  else
    render json: { errors: rent.errors.messages }, status: :unprocessable_entity
  end
end
```

Another thing to remark here is that you are allowing the user to create a Rent with any param he wants! That's a big security issue, you always have to whitelist all params!! [Here](http://edgeguides.rubyonrails.org/action_controller_overview.html#strong-parameters) is some reading about that, take a look at that please!

Last but not least, if I'm logged in as an user with ID=1, but I consume this endpoint sending `"user_id": 2` I will be creating a Rent for another user!! Devise Token Auth gives you a helper to access the current user (`current_user`), you can use that helper to fetch the ID of the user and create the rent associated with him. 
Rails allows you to do something magical like this
```ruby
rent = current_user.rents.build(create_params)
```
Which is exactly the same as
```ruby
rent = Rent.new(create_params.merge(user_id: current_user.id))
```